### ISSUE TITLE
long-term prediction

### ISSUE THREAD

**Issue by qsa-fox** [JannerM/trajectory-transformer#17]
Hi, how can I reproduce the Trajectory predictions in Figure 2 in the paper?
Does this long-term prediction also need discrete states and actions? Hope for more details about the long-term prediction experiment implementations.

**Comment by ghost** [comment]
Yes, it also requires discretization since the trajectory transformer only operates on discretized data.
Appendix E Beam Search Hyperparameters in the paper seems to only specify the parameters for planning.

It would be really helpful if the authors @jannerm could state the hyperparameters for this rollout generation.
Further questions are:
- What is the used rollout policy for the PETS feedforward gaussian dynamics model?
- Which dataset was used for the humanoid environment?

Best,
Niclas
